pkg/clients/nodepool/nodepool_test.go

```diff
@@ -558,26 +557,6 @@ func TestIsUpToDate(t *testing.T) {
 				isErr:    false,
 			},
 		},
-		"UpToDateIgnoreRefs": {
```

Comment:
I don't think these were actually doing anything anymore, as IsUpToDate doesn't seem to treat references specially.